applications/universaldot.me.md

```diff
@@ -0,0 +1,292 @@
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 3
+
+
+## Project Overview :page_facing_up:
+
+> *This application is a follow up grant to: https://github.com/w3f/Grants-Program/pull/714*
+
+
+### Overview
+
+Universaldot.me is a **freelancing decentralized application (dApp)** that allows individuals to create/complete tasks and organize themselves by creating geographically distributed organizations. As such, it intends to simplify the way people and organizations create contracts between one another. The layer of trust and transparency is provided by distributed blockchain infrastructure (**Substrate**). The foundation of the application has already been built with custom pallets functionality that provides the necessary business logic for creating a freelancing platform. This functionality needs to be further enriched with additional business logic, for which we are requesting funds to further build the solution.
+
+In addition to enriching the blockchain business logic, we intend to redesign parts of the User Interface with a completely new User Experience. The UI should provide seamless interaction with the web application. More specifically, the React components related to the DAO need to be redesigned and further developed.
+
+A major part of the next milestone also includes designing and implementing machine learning models using TensorFlow which will recommend tasks and organizations to individuals.
+
+Universaldot.me intends to become a Polkadot Parachain in the future.
+
+
+### Project Details
+
+
+
+![Architecture Design](https://github.com/UniversalDot/documents/blob/master/designs/architecture/3Tier_Architecturev3.png?raw=true) 
+Figure 1. Architecture Diagram
+
+- [Software Architecture Document](https://drive.google.com/file/d/13C9IRIX49AjfZEB-MI9dXG3gX-cdTgBp/view?usp=sharing) 
+
+
+- [Vision Document](https://github.com/UniversalDot/documents/blob/master/docs/UniversalDot_Vision.pdf) 
+
+- [Presentation](https://github.com/UniversalDot/documents/blob/master/presentations/UNIVERSAL%20DOT%20FOUNDATION%20Problem_Solution%20.pdf)
+
+- [Mockups/Preliminary Design](https://github.com/UniversalDot/documents/blob/master/designs/documents/UniversalDot%20Prototype.pdf)
+
+- [Software Requirements Document](https://drive.google.com/file/d/1HvYvb7N1A9uT_9UhrAq1mV38RtjyfVv-/view?usp=sharing)
+
+- [Milestone Definition Document](https://drive.google.com/file/d/1JUzlLnvOQ-SN3L4vk5sBUp39BWqUaM9G/view?usp=sharing)
+
+- [Substrate Builder Program Pitch Presentation](https://drive.google.com/file/d/1pHe0E-YREhfPSlzErKzU4XUpHvH2bF-q/view?usp=sharing)
+
+
+### Ecosystem Fit
+
+*Where and how does your project fit into the ecosystem?*
+- The dApp eventually aims to become a parachain on polkdot. 
+
+*Who is your target audience (parachain/dapp/wallet/UI developers, designers, your own user base, some dapp's userbase, yourself)?*
+- Users who want to create decentralized organizations and projects. 
```

Comment:
Yes, conceptually it is similar to decentralized Jira. Organizations can manage a Kanban Board of `Tasks`, and these tasks will be suggested to users based on their `Profile` information. 

In our current design, `Task` is very generically implemented. This means that a task can be something like:

- take me from place A to place B (example in car-sharing)
- create web design for our company. 
- create a decentralized Jira. 

The creator of the `Tasks` basically chooses the level of abstraction for his task. 

We see the `DAO` as a collection of `Tasks`, which fulfill the organizational vision. As such, for now, we don't intend to implement multiple projects inside a DAO. It might be needed in the future if the market demands this new abstraction.